src/index.css

```diff
@@ -33,6 +32,7 @@ body {
 }
 
 .profile {
+  margin-top: 10rem;
```

Comment:
"em" is the fontsize, so it lets you make things more dynamic. "rem" is the global font size. So you're basically saying, 10*global font size.